lms/djangoapps/student_account/views.py

```diff
@@ -347,7 +349,10 @@ def get_user_orders(user):
                                     'order_date': strftime_localized(
                                         date_placed.replace(tzinfo=pytz.UTC), 'SHORT_DATE'
                                     ),
-                                    'receipt_url': commerce_configuration.receipt_page + order['number']
+                                    'receipt_url': '{receipt_page_url}{order_number}'.format(
```

Comment:
Does it make sense to pass the order number to the receipt page URL and have that be responsible for the full construction?  It seems like this is a critical piece of data for the URL to actually be useful.